test/indexer/test_indexer_controller.py

```diff
@@ -0,0 +1,139 @@
+import json
+import logging
+
+import boto3
+import mock
+from moto import (
+    mock_sqs,
+    mock_sts,
+)
+
+from azul import (
+    config,
+    queues,
+)
+from azul.azulclient import (
+    AzulClient,
+)
+from azul.indexer import (
+    BundleFQID,
+)
+from azul.indexer.index_controller import (
+    IndexController,
+)
+from indexer import (
+    ForcedRefreshIndexService,
+    IndexerTestCase,
+)
+
+log = logging.getLogger(__name__)
+
+
+class Record:
+    body = None
+    _event_dict = None
+
+    def __init__(self, body):
+        self.body = body
+        self._event_dict = dict(attributes=dict(ApproximateReceiveCount=1))
+
+    def to_dict(self):
+        return self._event_dict
+
+
+class IndexControllerTestCase(IndexerTestCase):
+
+    def setUp(self) -> None:
+        super().setUp()
+        self.index_service.create_indices(self.catalog)
+        self.client = AzulClient()
+        self.controller = IndexController()
+
+    def tearDown(self):
+        self.index_service.delete_indices(self.catalog)
+        super().tearDown()
+
+    def generate_events(self):
```

Comment:
What's this?

Reply:
It was a method that assisted with event generation, its been refactored to assist with creation of SQSRecord objects.

Reply:
it was refactored  see https://github.com/DataBiosphere/azul/blob/e6b3a14a9624c24977675de3a52571c3e6a4be52/test/indexer/test_indexer_controller.py#L64

Reply:
that was stupidity on my part, i did not see that the method was still present. Its been removed. 😬